templates/bot/node/load/index.js

```diff
@@ -0,0 +1,22 @@
+process.env.LEO_ENVIRONMENT = "test";
```

Comment:
This doesn't export a handler

Reply:
I could change this to support lambda, but I wasn't sure since loading commonly occurs many different ways.